src/d3a/models/strategy/external_strategies/__init__.py

```diff
@@ -137,6 +152,10 @@ def market_area(self):
     def device(self):
         return self.owner
 
+    @property
```

Comment:
Can you please tell me the reason why it is better to declare it as a property here? Thanks

Reply:
I think the property makes sense here, in order to conveniently access the communicator class without having to always do `self.owner.config.external_redis_communicator.publish()` for instance. An alternative would be to simply store the reference to the object of course (which would save some CPU cycles for looking up over these area members, but the savings will be minuscule), but to my eyes this solution also works. 
If the concern is why to not make it a function, it is to save the explicit () that you will have to add otherwise (essentially one would have to do self.redis().publish(), contrary to self.redis.publish() which is needed with this implementation)

Reply:
Well the main reason was in setup file I have to explicitly pass `config` as a parameter to all areas else they are considering the default `GlobalConfig` from here `src/d3a/models/area/__init__.py:253` which doesn't have `external_redis_communicator` BUT once all the areas have been activated  `config` obj is respectively passed to all area and `GlobalConfig` not being used later